packages/client/src/route/public.tsx

```diff
@@ -66,7 +64,10 @@ function RouterComp({ route }: { route: string }) {
     // The client and auth settigns will not be needed on these routes
     if (!/auth\/oauth/.test(location.pathname)) {
       AuthService.doLoginAuto()
-      AuthSettingsService.fetchAuthSetting()
+      waitForClientAuthenticated().then(async () => {
+        const authenticationSettingsResponse = await Engine.instance.api.service('authentication-setting').find()
+        authSettingsState.set(authenticationSettingsResponse.data)
```

Comment:
since we arent using the actual response, we can avoid this second `'authentication-setting'` call and probably the waitForClientAuthenticated() altogether by using AuthState.isLoggedIn

_app.tsx or main.tsx should handle this